proto/umee/ugov/v1/query.proto

```diff
@@ -34,6 +36,22 @@ service Query {
   rpc InflationCycleEnd(QueryInflationCycleEnd) returns (QueryInflationCycleEndResponse) {
     option (google.api.http).get = "/umee/ugov/v1/inflation_cycle_end";
   }
+
+  // DenomOwners queries for all account addresses that own a particular token
+  // denomination.
+  rpc DenomOwners(QueryDenomOwners) returns (cosmos.bank.v1beta1.QueryDenomOwnersResponse){
+    option (google.api.http).get = "/umee/ugov/v1/denom_owners";
```

Comment:
Let's try to add height option here.
Example:
```
option (google.api.http).get               = "/umee/uibc/v1/denom_owners/{height}";
```

Reply:
height will be work with grpc_query
Link: https://docs.cosmos.network/v0.50/user/run-node/interact-node#query-for-historical-state-using-grpcurl